modules/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 # ---[ Add modules
+# With build_windows_shared.bat file detectron ops build and work on MSVC 14 toolset
+option(BUILD_DETECTRON_WITH_MSVC "This option is set to ON with build_windows_shared bat file" OFF)
 # TODO(orionr): Enable Detectron ops for Windows DLL when we
 # can figure out how to get it to build
-if (NOT (MSVC AND BUILD_SHARED_LIBS))
+if ((NOT (MSVC AND BUILD_SHARED_LIBS)) OR BUILD_DETECTRON_WITH_MSVC)
```

Comment:
What about just removing `NOT MSVC`?

Reply:
Fixed it.

BUILD_DETECTRON_WITH_MSVC option was added, because I was thinking there is some reason, that I don't see, why 
if ((NOT (MSVC AND BUILD_SHARED_LIBS)))
initially was there. 

As there is no real reason, it is removed now.